app/views/layouts/mailer.html.haml

```diff
@@ -1,7 +1,8 @@
 !!!
 %html
   %head
-    %meta{:content => "text/html; charset=utf-8", "http-equiv" => "Content-Type"}/
+    %meta
+      { :content => "text/html; charset=utf-8", "http-equiv" => "Content-Type" }/
```

Comment:
What was wrong with this one out of interest? Line too long?

Reply:
It was the missing spaces after the opening curly brace and before the closing. I've just rechecked and now with the latest Rubocop version, it has changed this line again, but differently.